core/storage/question/gae_models.py

```diff
@@ -537,7 +537,7 @@ class QuestionSummaryModel(base_models.BaseModel):
     The key of each instance is the question id.
     """
     # The user ID of the creator of the question.
-    creator_id = ndb.StringProperty(required=True)
+    creator_id = ndb.StringProperty(required=True, indexed=True)
```

Comment:
Are we expecting to have an automated change in index.yaml file for this?

Reply:
These simple indexes are not in index.yaml but need the migration.